[abort-controller] prepare for 1.0.1 release (#6412)
Azure/azure-sdk-for-js: sdk/core/abort-controller/changelog.md

## 1.0.1 - 2019-12-04

> Fixes an issue that can occur with angular prod builds due to triple-slash directives.

Reviewer: ```suggestion
Fixes the [bug 6271](https://github.com/Azure/azure-sdk-for-js/pull/6271) that can occur with angular prod builds due to triple-slash directives.
```

> Fixes the [bug 6271](https://github.com/Azure/azure-sdk-for-js/pull/6271) that can occur with angular prod builds due to triple-slash directives.

Reviewer: ```suggestion
Fixes the [bug 6271](https://github.com/Azure/azure-sdk-for-js/issues/6271) that can occur with angular prod builds due to triple-slash directives.
```

> ([6344](https://github.com/Azure/azure-sdk-for-js/pull/6344))

Reviewer: ```suggestion
([PR 6344](https://github.com/Azure/azure-sdk-for-js/pull/6344))
```
